src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -16,16 +16,7 @@
              MouseRightButtonDown="topControl_MouseRightButtonDown"
              MouseEnter="OnNodeViewMouseEnter"
              MouseLeave="OnNodeViewMouseLeave">
-    <UserControl.Resources>
```

Comment:
Removed this to control the opacity only for the node. Otherwise preview bubble looks different.